### PR TITLE
Style Polygon community on Peeranha

### PR DESCRIPTION
**Pull request by Dmitrijg24k**
Styled Community Polygon on Peeranha in the communities-configs folder.
Roboto font was used.

### CODE REVIEW

File: server/port.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 const argv = require('./argv');
 
-module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
```
Comment (ruslanzharkov) [inline server/port.js:3]: please do a rollback

File: app/communities-configs/polygon/images/polygon-logo.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...0055 18.0296L4.61393 15.5941C3.9548 15.1882 3.4934 14.444 3.4934 13.6321V8.82875C3.4934 8.01691 3.88888 7.27273 4.61393 6.86681L8.63464 4.49894C9.29377 4.09302 10.1506 4.09302 10.8757 4.49894L14.8964 6.86681C15.5555 7.27273 16.0169 8.01691 16.0169 8.82875V11.9408L19.5103 9.84355V6.7315C19.5103 5.91966 19.1149 5.17548 18.3898 4.76956L10.9416 0.30444C10.2825 -0.10148 9.4256 -0.10148 8.70055 0.30444L1.12053 4.76956C0.39548 5.17548 0 5.91966 0 6.7315V15.7294C0 16.5412 0.39548 17.2854 1.12053 17.6913L8.70055 22.1564C9.35969 22.5624 10.2166 22.5624 10.9416 22.1564L16.0828 19.112L19.5762 17.0148L24.7175 13.9704C25.3766 13.5645 26.2335 13.5645 26.9585 13.9704L30.9792 16.3383C31.6384 16.7442 32.0998 17.4884 32.0998 18.3002V23.1036C32.0998 23.9154 31.7043 24.6596 30.9792 25.0655L26.9585 27.5011C26.2994 27.907 25.4425 27.907 24.7175 27.5011L20.6968 25.1332C20.0376 24.7273 19.5762 23.9831 19.5762 23.1712V20.0592L16.0828 22.1564V25.2685C16.0828 26.0803 16.4783 26.8245 17.2034 27.2304L24.7834 31.6956C25.4425 32.1015 26.2994 32.1015 27.0244 31.6956L34.6045 27.2304C35.2636 26.8245 35.725 26.0803 35.725 25.2685V16.2706C35.725 15.4588 35.3295 14.7146 34.6045 14.3087L26.9585 9.7759Z" fill="#7950DD"/>
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/images/polygon-logo.svg:2]: please try to optimize svg via this tool
https://jakearchibald.github.io/svgomg/
screenshot with optimization would be great

File: app/communities-configs/polygon/index.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+const domainName = 'https://polygon.technology/';
+const domainDocs = 'https://docs.polygon.technology/';
+const domainBlog = 'https://blog.polygon.technology/';
+
+export const CustomSubHeaderConfig = {
+  design: 'telos_style',
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:19]: `telosStyle`, camelCase style

File: app/communities-configs/polygon/index.js [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        padding: 0 !important;
+
+        background: transparent !important;
+      }
+    `,
+  },
+  links: [
+    {
+      text: 'Scaling Solutions',
+      // href: `${domainName}dev`,
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:82]: why is it commented? should be removed if unnecessary

File: app/communities-configs/polygon/index.js [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+const domainName = 'https://polygon.technology/';
+const domainDocs = 'https://docs.polygon.technology/';
+const domainBlog = 'https://blog.polygon.technology/';
+
+export const CustomSubHeaderConfig = {
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:18]: `customSubHeaderConfig`

File: app/communities-configs/polygon/index.js [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+      color: rgba(255,255,255,1) !important;
+      transition: background .22s ease-in-out;
+      border-radius: 300px;
+
+      :hover {
+        border-color: rgba(100,51,185,1)
+        background-color: rgba(100,51,185,1)
+      }
+    `,
+    subitems: `
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:56]: use `css` from styled-component, check other places

File: app/communities-configs/polygon/index.js [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+        left: 5px;
+        top: -5px;
+
+        padding: 0 !important;
+
+        background: transparent !important;
+      }
+    `,
+  },
+  links: [
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:79]: array with links might be moved to own variable

File: app/communities-configs/polygon/index.js [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+import favicon from '!file-loader?name=[name].[ext]!images/favicon-telos.ico';
+import PolygonLogo from './images/polygon-logo.svg?inline';
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+const domainName = 'https://polygon.technology/';
+const domainDocs = 'https://docs.polygon.technology/';
+const domainBlog = 'https://blog.polygon.technology/';
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:16]: this 3 constants should be extracted from this file to some url config file or something else

File: app/communities-config.js [hunk 9]
```diff
@@ -57,8 +59,10 @@ const communitiesConfig = {
     // },
   },
   dev: {
-    3: {
+    2: {
       origin: 'http://localhost:31000',
```
Comment (ruslanzharkov) [inline app/communities-config.js:63]: why origin is localhost with this port?

Comment (ruslanzharkov) [inline app/communities-config.js:63]: why origin is localhost with this port?

File: app/communities-configs/polygon/index.js [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+    h3LetterSpacing: '0.5px',
+    tagsLetterSpacing: '0.5px',
+    questionTitleLetterSpacing: '0.5px',
+    h3: 'Roboto-Regular, Arial, sans-serif',
+    main: 'Roboto-Regular, Arial, sans-serif',
+    questionItemTitle: 'Roboto-Regular, Arial, sans-serif',
+    questionTitleFont: 'Roboto-Regular, Arial, sans-serif',
+  },
+  colors: {
+    blue: '#5D6DFE',
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:160]: colors should be added to `style-constants`, also you can reuse some of this constants and not producing the same colors

File: app/communities-configs/polygon/index.js (hunk 6 above)
Comment (ruslanzharkov) [reply app/communities-configs/polygon/index.js:56]: also check other places with styles

File: app/communities-configs/polygon/index.js [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+    bg: {
+      header: 'rgba(255,255,255,1)',
+      dropdown: '#ffffff',
+    },
+    color: {
+      a: 'rgba(41,41,52,1)',
+      arrow: 'rgba(238,238,242,1)',
+    },
+    font: {
+      body: 'Roboto-Regular, Arial, sans-serif',
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:30]: `Roboto-Regular, Arial, sans-serif` repeats 5 times in this file, could you please create a variable for this and reuse it everywhere?

File: app/communities-config.js (hunk 9 above)
Comment (YuriyLu) [reply app/communities-config.js:63]: it should just be a random port for the dev environment, isn't it?

Comment (ruslanzharkov) [reply app/communities-config.js:63]: If it's only for dev purposes looks ok to me, just wondering

File: app/communities-configs/polygon/index.js (hunk 10 above)
Comment (Dmitrijg24k) [reply app/communities-configs/polygon/index.js:160]: As far as I understand, lodash/get in style-constants.js takes color constants from array 'colors' in index.js

File: app/communities-configs/polygon/images/polygon-logo.svg (hunk 2 above)
Comment (Dmitrijg24k) [reply app/communities-configs/polygon/images/polygon-logo.svg]: jakearchibald github

File: app/communities-configs/polygon/index.js (hunk 10 above)
Comment (ruslanzharkov) [reply app/communities-configs/polygon/index.js:165]: You can introduce your new colors without using lodash/get functionality, it's better to control colors when they are located in one place, style-constants looks good for this.

File: app/communities-configs/polygon/index.js [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+import robotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import robotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+import {
+  domainName,
+  domainDocs,
+  domainBlog,
+} from './urls'
+
+
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:20]: remove empty line

File: app/communities-configs/polygon/index.js (hunk 10 above)
Comment (Dmitrijg24k) [reply app/communities-configs/polygon/index.js:227]: Wouldn't the next chain work in this case?
style-constants 'red' -> import red from style-constants in index.js -> get red from index.js in style-constants.
And yes if I use these colors later, I should specify them in style-constants, but these colors are used only for this community.

File: app/communities-configs/polygon/index.js [hunk 13]
```diff
@@ -1,55 +1,70 @@
...
+import peeranhaPoS from './images/polygon-pos.svg?inline';
 import {
   domainName,
   domainDocs,
   domainBlog,
-} from './urls'
-
-
-const fonts = 'Roboto-Regular, Arial, sans-serif';
+} from './urls';
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:13]: empty line after import

File: app/communities-configs/polygon/index.js [hunk 14]
```diff
@@ -95,51 +109,99 @@ export const customSubHeaderConfig = {
     },
     CustomSubHeader: css`
       padding: 20px 0;
-      background: radial-gradient(66.32% 66.32% at 54.13% 113.95%,rgba(108,38,255,.2) 0,rgba(242,89,255,0) 100%),linear-gradient(211.99deg,rgba(65,157,241,.2) -4.17%,rgba(45,143,234,0) 68.7%),radial-gradient(100% 100% at 28.65% 0,rgba(109,0,255,.25) 0,rgba(250,247,254,0) 100%);
+      background: '#ffffff';
```
Comment (ruslanzharkov) [inline app/communities-configs/polygon/index.js:112]: import #ffffff form `style-constants.js`

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 15]
```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import communitiesConfig from 'communities-config';
+import {
+  isSingleCommunityWebsite,
+  singleCommunityStyles,
+} from 'utils/communityManagement';
+import peeranhaLogo from 'images/LogoBlack.svg?inline';
+import polygonPoSLogo from '../../../communities-configs/polygon/images/polygon-pos.svg?inline';
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:10]: please use absolute path

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+import polygonEdgeLogo from '../../../communities-configs/polygon/images/polygon-edge.svg?inline';
+import polygonAvailLogo from '../../../communities-configs/polygon/images/polygon-avail.svg?inline';
+import polygonZeloLogo from '../../../communities-configs/polygon/images/polygon-zero.svg?inline';
+import polygonMidenLogo from '../../../communities-configs/polygon/images/polygon-miden.svg?inline';
+import polygonHermezLogo from '../../../communities-configs/polygon/images/polygon-hermez.svg?inline';
+import polygonNightfallLogo from '../../../communities-configs/polygon/images/polygon-nightfall.svg?inline';
+import { LocalLink, Subitems } from '../CustomSubHeader';
+const logosSVG = {
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:18]: empty line after import

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+  'polygonPoS': polygonPoSLogo,
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:19]: remove quotes for all keys

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+  'polygonNightfall': polygonNightfallLogo,
+}
+const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.5rem;
+  margin: 0px 2rem;
+}
+`;
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:34]: add empty line after this line

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 19]
```diff
@@ -0,0 +1,165 @@
...
+const logosSVG = {
+  'polygonPoS': polygonPoSLogo,
+  'polygonEdge': polygonEdgeLogo,
+  'polygonAvail': polygonAvailLogo,
+  'polygonZero': polygonZeloLogo,
+  'polygonMiden': polygonMidenLogo,
+  'polygonHermez': polygonHermezLogo,
+  'polygonNightfall': polygonNightfallLogo,
+}
+const Container = styled.div`
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:27]: empty line

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx (hunk 18 above)
Comment (ruslanzharkov) [reply app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:34]: and please check for all places in file

File: app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx [hunk 20]
```diff
@@ -0,0 +1,165 @@
...
+  justify-content: flex-center;
+  > div {
+    display: flex;
+    align-items: center;
+    > div {
+      padding: 0;
+      line-height: 1.25;
+    }
+    > div:hover {
+      a{
```
Comment (ruslanzharkov) [inline app/components/CustomSubHeaders/CustomSubHeaderContainers/PolygonStyleTopNav.jsx:72]: `a {`

**Review by ruslanzharkov — APPROVED** [review]

File: app/communities-configs/polygon/index.js (hunk 10 above)
Comment (ruslanzharkov) [reply app/communities-configs/polygon/index.js:246]: Ok let's keep it as it is

**Pull request merged**